python/tests/pyspark/lib/assertions.py

```diff
@@ -0,0 +1,35 @@
+import re
+from pandas.testing import assert_frame_equal
+
+
+def _print_dfs(actual, expected):
+    print('actual.show(100):')
+    actual.show(100, False)
+    print('expected.show(100):')
+    expected.show(100, False)
+
+
+def assert_df(actual, expected, sort=True, **kwargs):
```

Comment:
kwargs are generally dangerous and hard to document. Which kwargs does assert_df accepts ? By reading the function code I think they are the same kwargs as assert_frame_equal but this is not immediate from the function signature

Reply:
Yes, those are simply passed on to assert_frame_equal. That should be added to docstring here.

I think passing kwargs is more future-proof than copying all of these and trying to keep them up-to-date:

```
def assert_frame_equal(left, right, check_dtype=True,
                       check_index_type='equiv',
                       check_column_type='equiv',
                       check_frame_type=True,
                       check_less_precise=False,
                       check_names=True,
                       by_blocks=False,
                       check_exact=False,
                       check_datetimelike_compat=False,
                       check_categorical=True,
                       check_like=False,
                       obj='DataFrame'):
```

On the other hand as long as nothing needs to be passed kwargs can be just removed.